### DISCUSSION TITLE
O que achou do Forum?

### DISCUSSION THREAD

**Discussion by jacobaraujo7** [Flutterando/forum#2] [Votações]
Aqui temos um exemplo de como funcionaram as futuras votações sobre o assunto da comunidade.

**Comment by tecrodrigocastro** [comment]
Ótima ideia !

**Comment by ramonsantospinto** [comment]
Achei maravilhoso.. Bem organizado , objetivo.

**Comment by GabrielcDonato** [comment]
Bem intuitivo e organizado

**Comment by JoaoWitorFelipe** [comment]
Boa ideia achei mais organizado e objetivo do que no Telegram

**Comment by seadramdesenv** [comment]
Ideia muito boa, no discord e telegram é muito bom, porém as vezes a duvida sobe pois tem muita interação, ai se perde, porém por aqui fica cada uma registrada separadamente... adorei

**Comment by correiarangel** [comment]
Parabéns mais uma ferramenta que será de muita utilidade além de documentar e arquivar as respostas de forma organizada.

**Comment by jair-araujo** [comment]
Acredito que será de grande ajuda, ter essas informações em um local de fácil consulta, principalmente para quem está iniciando em Flutter assim como eu, tirar duvidas é de ótima oportunidade. Além de poder adquirir conhecimento e compartilhar de quem já está fomentando o cenário de Flutter no Brasil.

**Comment by luizaraujodev** [comment]
Ótima ideia! Certeza que vai dar certo!

**Comment by ewertonls** [comment]
Eh bom para se tirar duvidas e fazer perguntas, mas como interaçao de comunidade eu acho mais interessante o discord ou telegram justamente por ser temporal, aqui a ideia eh mais conteudo e referencia de duvidas, ideias e sugestoes.

**Comment by GabrielCR99** [comment]
É muito boa mesmo! Quase que um "Dart/Flutter Overflow". Meu único receio é que, caso surjam dúvidas de coisas mais específicas do framework (como alguns packages famosos), a galera caia em cima, com um hate gratuito. Acho interessante ficar de olho pra que isso não aconteça igual em certas comunidades

**Comment by joasnog** [comment]
Acompanhando por aqui. Mt legal

**Comment by marcoswlrich** [comment]
Começando em Dart agora. Esse fórum foi uma ótima ideia. Parabéns

**Comment by fernandodeabreupereira** [comment]
Muito legal, bem organizado.

**Comment by ericqueirozc** [comment]
Acredito que a forma que a estrutura de fórum deixa registrado a discussão facilita uma consulta futura quando um erro parecido acontecer conosco.
Permite também que discussões se aprofundem mais já que mensagens de tópicos diferentes não vão aparecer no meio da time line e tirar o foco do assunto. 
Acho uma adição muita boa para a comunidade!

**Comment by KaiserGid** [comment]
ótimo, pois fica fácil de achar conteúdo q se perdem em meio a muitas
 msgs.

**Comment by matheusrmribeiro** [comment]
Ideia muito boa!
Eu já costumo ajudar a galerinha no StackoverflowBR, mas vejo muita gente não curte perguntar por lá por conta de uma galera chata que tudo bloqueia as perguntas "por acharem que a maioria fere as métricas do site", então vou começar a tentar ajudar por aqui também.
Um ponto muito positivo é que aqui da pra organizar melhor as perguntas e respostas, do que em um chat do Telegram que acaba se perdendo o contexto muitas vezes. Sem contar o markdown que ajuda muito.

**Comment by renanzdm** [comment]
É muito bacana

**Comment by fredfv** [comment]
Muito boa a proposta!

**Comment by arqeco** [comment]
Parabéns pela iniciativa.

Há alguns meses propus criarmos um Reddit para Flutter em português. Num dos grupos do Telegram ninguém me respondeu, e no outro fui acusado de tentar dividir a comunidade.

Fiquei pasmo com isso, pois meu objetivo era **exatamente o mesmo** de vocês ao criarem esse fórum aqui: Manter um lugar onde as discussões fossem organizadas por tópicos, e que o conteúdo não se perdesse no fluxo contínuo e desorganizado dos aplicativos de mensagens.

Me desculpe se não entendi direito. Por favor me esclareçam. As discussões vão acontecer diretamente aqui, ou os conteúdos dos grupos é que vai ser copiado e colado aqui?

Saudações,
Márcio

**Comment by Dreyfi** [comment]
Sensacional, bem mais organizado que o discord.
